console-extensions.json

```diff
@@ -16,5 +16,11 @@
       "perspective": "admin",
       "section": "home"
     }
+  },
+    {
```

Comment:
This would be better to add to the demo plugin, then we can test the extension point in console CI. The template is mean to be a minimal plugin to use as a starting point. I'd expect most plugins won't need this particular extension.